CHANGELOG.latest.md

```diff
@@ -1,9 +1,55 @@
+* Update Hybrid Common 4.1.0 and add AdServices (#167) via Josh Holtz (@joshdholtz)
+* Update deployment (#166) via Joshua Liebowitz (@taquitos)
+* Preparing for version 3.0.0-rc.4 (#165) via Joshua Liebowitz (@taquitos)
+* Rename remaining snake_case items to camelCase (#164) via Joshua Liebowitz (@taquitos)
+* Documentation generation (#159) via NachoSoto (@NachoSoto)
+* `README`: fixed minimum Xcode / iOS version (#160) via NachoSoto (@NachoSoto)
+* Prepare for 3.0.0-rc.3 (#158) via Joshua Liebowitz (@taquitos)
+* Update Purchases.setup to Purchases.configure (#157) via Joshua Liebowitz (@taquitos)
+* Preparing for version 3.0.0-rc.2 (#155) via Joshua Liebowitz (@taquitos)
+* getProductInfo renamed getProducts (#156) via Joshua Liebowitz (@taquitos)
+* Update data model for iOS RCv4, Android v5 (#154) via Joshua Liebowitz (@taquitos)
+* Update VERSIONS.md (#153) via Joshua Liebowitz (@taquitos)
+* Preparing for version 3.0.0-rc.1 (#148) via Joshua Liebowitz (@taquitos)
+* Adds cordova-annotated-plugin-android dependency to package.json (#152) via Cesar de la Vega (@vegaro)
+* Final cleanup before prerelease (#147) via Joshua Liebowitz (@taquitos)
+* Bump PHC to 3.2.2 (#146) via Joshua Liebowitz (@taquitos)
+* Phase 1 of update/add/remove api (#145) via Joshua Liebowitz (@taquitos)
+* Group logic together in extensions (#143) via Joshua Liebowitz (@taquitos)
+* Update Purchaser/Customer related naming -> Rename of old API before adding in new API. (#141) via Joshua Liebowitz (@taquitos)
+* Update project setup script and Swift to 5.0 (#140) via Joshua Liebowitz (@taquitos)
+* Update to most phc 3.2.1, add iOS troubleshooting instructions (#139) via Joshua Liebowitz (@taquitos)
+* iOS: Add "Refresh js resources" build phase run script (#138) via Joshua Liebowitz (@taquitos)
+* Android: Adopt purchases-hybrid-common:3.2.0 (#137) via Joshua Liebowitz (@taquitos)
+* Adopt PurchasesHybridCommon 3.2.0 (#136) via Joshua Liebowitz (@taquitos)
+* Swift Migration (#134) via Joshua Liebowitz (@taquitos)
+* Add StoreKit config file to xcproj (#135) via Joshua Liebowitz (@taquitos)
+* Automatically add StoreKit config into project (#133) via Joshua Liebowitz (@taquitos)
+* Optional API key for setup.bin (#132) via Joshua Liebowitz (@taquitos)
+* Dependency broke us, update TypeScript (#131) via Joshua Liebowitz (@taquitos)
+* Update willRenew comment - will be false for lifetime (#129) via Josh Holtz (@joshdholtz)
+* Remove issue bot (#126) via beylmk (@beylmk)
+* Adds more exempt labels to stale bot (#125) via Cody Kerns (@codykerns)
+* Update xcode for Android tests, rename Flutter to Cordova in description, uncomment out Android workflow so it runs (#123) via Josh Holtz (@joshdholtz)
+* [github] update issue templates with emojis, same feature request url (#118) via Josh Holtz (@joshdholtz)
+* bot updates (#116) via Jazmine McGinnis (@JazmineMT)
+* unversion `package-lock.json` (#115) via aboedo (@aboedo)
```

Comment:
Seems like a huge changelog, not sure if we should minimize it

Reply:
I just pushed a change that simplifies it. Let me know what you think